### DISCUSSION TITLE
Loading Json files

### DISCUSSION THREAD

**Discussion by phred1** [tiboun/python-bigquery-test-kit#28] [Q&A]
Hi, I am trying to use your package, which looks quite useful I must say! 😄 

However, when trying to load a json file with the `PackageFileLoader`, I keep getting
`ModuleNotFoundError` out of it. I tried to provide multiple different paths, but I keep getting the error. 

Here is my file structure:
```
- test_BigQuery.py
- data/
      - newline_delimited_data.json
```
     
In `test_BigQuery.py` I have:

`pfl = PackageFileLoader("data/newline_delimited_data.json")
`

Has this happened to someone in the past?
Thanks in advance for the help!

**Comment by matlavoie-gp** [comment]
I think there is a bug in the PackageFileLoader.  It assumes the path is a package and does this.
        self.package = dirname(path).replace("/", ".").replace("\\", ".")
